.github/workflows/release.yml

```diff
@@ -235,22 +260,21 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-pip-
       - uses: actions/setup-dotnet@v1
-        if: ${{ matrix.language == 'dotnet' }}
+        if: ${{ needs.get-smoke-test-configuration.outputs.language == 'dotnet' }}
         with:
           dotnet-version: '3.1.x'
       - name: download layer tf file
         uses: actions/download-artifact@v2
         with:
           name: layer.tf
       - name: overwrite layer.tf
-        if: ${{ github.event.inputs.layer_kind == matrix.layer_kind }}
         run: |
-          cat ${{ matrix.terraform_directory }}/layer.tf
-          mv -f layer.tf ${{ matrix.terraform_directory }}/layer.tf
-          cat ${{ matrix.terraform_directory }}/layer.tf
+          cat ${{ needs.get-smoke-test-configuration.outputs.terraform_directory }}/layer.tf
+          mv -f layer.tf ${{ needs.get-smoke-test-configuration.outputs.terraform_directory }}/layer.tf
+          cat ${{ needs.get-smoke-test-configuration.outputs.terraform_directory }}/layer.tf
       - name: Build functions
-        run: ${{ matrix.build_command }}
-        working-directory: ${{ matrix.build_directory }}
+        run: eval ${{ needs.get-smoke-test-configuration.outputs.build_command }}
```

Comment:
I saw online that there are [some code injection security risks with using `eval`](https://stackoverflow.com/questions/5615717/how-to-store-a-command-in-a-variable-in-a-shell-script) but I think it should be okay in our environment. Let me know if anyone thinks this is an issue!

Reply:
I think normally you use `sh -c` instead of eval. But IIUC, the only command that wouldn't work directly is python since it has the `cd sample_apps` - is it possible to move that into the build script or somehow make it one command just like the others? Node also, even a one line `build.sh` so it's all consistent would be good

Reply:
Yes you're right that `python38` and `nodejs` were the exceptions that required `eval`. Thanks for your tip, I realized we can change the directory for `python38` and fix it that way. For `nodejs`, it looks like there is already a `build.sh` so I didn't want to replace or edit it. Instead, I created a new `./build_upstream.sh` to serve the same purpose. This way we don't need `eval`.